### ISSUE TITLE
Data simulation and augmentation

### ISSUE THREAD

**Issue by v-nhandt21** [rishikksh20/hifigan-denoiser#5]
Can you detail the way you are using to make the noise audio for training?

Does it the same with described in the paper?

![image](https://user-images.githubusercontent.com/36092323/147208742-4893bdd5-cd79-446f-ae42-24f0dba714ac.png)

Are you using kaldi or any tool for this, and can you share your noise dataset !

Thank rishikksh !

**Comment by LXnn058** [comment]
Hi, have you solved this problem?

**Comment by v-nhandt21** [comment]
> Hi, have you solved this problem?

With my experiment, the data augmented with Kaldi give better results than some methods that augment on the fly

**Comment by LXnn058** [comment]
Are the parameters the same as those set in the paper? Can you provide me the corresponding processing script? thanks.

**Comment by v-nhandt21** [comment]
I think the parameter depending on the test set / practical problem you want to solve. 

My config is SNR between 10dB and 30dB, use MUSAN as additive noise

You can check out the public source code here: https://github.com/zhaoyi2/audio_augment

**Comment by LXnn058** [comment]
This helped me a lot, thanks！

**Comment by skol101** [comment]
@v-nhandt21 I simply added audiomentations (RoomSimulator, Backgroundnoise) and stored to separate directory, which I use in "input_wavs" params after step for postnet training (250k in my case, though it's 500k in the default config).

**Comment by v-nhandt21** [comment]
> @v-nhandt21 I simply added audiomentations (RoomSimulator, Backgroundnoise) and stored to separate directory, which I use in "input_wavs" params after step for postnet training (250k in my case, though it's 500k in the default config).

Yep, it seems hard for the model to converge on the fly augmentation

**Comment by velonica0** [comment]
> > Hi, have you solved this problem?
> 
> With my experiment, the data augmented with Kaldi give better results than some methods that augment on the fly

Hi, Bro
Can I get a copy of your processed training data about this code, I don't understand the specific definitions of the folders and file trees in the code, such as gt_noise, gt_clean, generated, etc.
